src/parsers/manifestjson.js

```diff
@@ -373,14 +394,12 @@ export default class ManifestJSONParser extends JSONParser {
         );
       }
       if (this.parsedJSON.background.service_worker) {
-        // TODO: replace this with a call to this.validateFileExistsInPackage
-        // once background.service_worker is supported on the Firefox side and
-        // we can accept it on AMO submissions
-        // (https://github.com/mozilla/addons-linter/issues/3290).
```

Comment:
🏆

Reply:
This reminded me that this PR should mention #3290 (and I think also close it, because those steps would become implicit when this lands).

I'm not marking this comment as resolved yet, because I want to double-check if there was any other reference to #3290 around the sources and see if there is anything left to do from what #3290 does currently describe (eventually filing a separate issue if necessary).

Reply:
I double-checked and the same issue doesn't seem to be referenced anywhere else.